collection-scripts/gather_monitoring

```diff
@@ -125,6 +142,8 @@ monitoring_gather(){
 
   alertmanager_get status  || true
 
+  prom_query "sum(cnv:vmi_status_running:count)" running_vms_count || true
```

Comment:
We might want to add a prefix like:
cnv_running_vms_count 
so it will be clear who created the file and if we add additional metrics they will be sorted together.

Can you please show an example of how the metric will be printed in the file?

Reply:
Other example than the one in the description?